frontendcomponent/androidcomponent/src/main/res/values/text_apperance.xml

```diff
@@ -58,6 +58,14 @@
         <item name="android:textColor">@color/gray1</item>
     </style>
 
+    <!-- Notosans Regular-Bold 24sp -->
+    <style name="TextAppearance.NotoSans.Headline5" parent="TextAppearance.MaterialComponents.Headline5">
```

Comment:
Sorry, We don't want to create "TextAppearance.NotoSans".
Because of TextAppearance.App is that.
Please use TextAppearance.App.Headline5 and use bold in layout xml 🙏